src/ecs/registry.zig

```diff
@@ -216,7 +216,7 @@ pub const Registry = struct {
     pub fn assure(self: *Registry, comptime T: type) *Storage(T) {
         var type_id = utils.typeId(T);
         if (self.components.getEntry(type_id)) |kv| {
-            return @intToPtr(*Storage(T), kv.value);
```

Comment:
@leroycep If you haven't found it yet, there exists the same needed update in the dispatcher.zig file as well on line 21
```var signal = @intToPtr(*Signal(void), ptr.value_ptr.*);```

Reply:
Thanks! I've gone ahead and fixed this in the latest commits